### ISSUE TITLE
[Nightly 20170301] cannot even login ("Konnte den Host nicht finden")

### ISSUE THREAD

**Issue by Wikinaut** [nextcloud/android#702]
### Actual behaviour

Adding the server url is fine ("Sichere Verbindung hergestellt"), but after adding correct username and correct PIN-App (or the Passwort with disabled 2-Factor Authentification app, even with disabled fail2ban) there is then, when I click "Verbinden" the message

"Konnte den Host nicht finden"

### Environment data
Android version: Android 7.0
Device model: Huawei P8 Lite 2017
Stock or customized system: Stock
Nextcloud app version: Nextcloud Nightly 20170301
Nextcloud server version: Nextcloud 11.0.2
Installed Apps: Two Factor TOTP Provider 1.1.0 (and others)

### Logs
```
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2017-03-02T00:58:06+0100
Debug	no app in context	Accessing the API class is deprecated! Use the appropriate services instead!
```

**Comment by Wikinaut** [comment]
@AndyScherzinger Hi, can you please expressly confirm that the current beta (or the last nightly build) really works when it is newly installed ?

I found that it does not work for me (see issue report), and it may be that you have not tested recently the initial installation (installation and enter of server url, username and password or App PIN).

Just before you ask: yes, the fresh installation of 1.4.1 release version does not make any problem. I now went three times through this cycle (trying the beta, re-installing the release) and am frustrated.

@AndyScherzinger please do me a favour and try to fresh-install a beta with entering credentials.

**Comment by JSoko** [comment]
I installed 20170301 yesterday after deinstalling 20170228 and entering credentials worked for me.
Perhaps you'll have a look in the bruteforcetable of your database?

**Comment by Wikinaut** [comment]
@JSoko Äh, yes, good catch! (I was one of the protagonists of brute-force prevention, and the change to nextcloud was important in that sense....). I will check this tonight, and re-open this only if the problem persists. I remember, that clearing the table can become difficult, I mean, not everything is cleared automatically.

tl,dr;
I reported that as a bug, but perhaps it isn't. Closing it for now.
